docs/UserGuide.adoc

```diff
@@ -62,26 +65,34 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 
 Format: `help`
 
-=== Adding a person: `add` / `a`
+=== Adding a person: `add` / `a` kbd:[V2.0]
 //tag::bday[]
 Adds a person to the address book +
-Format: `add n/NAME [p/PHONE] [e/EMAIL] [a/ADDRESS] [b/BIRTHDAY](DD/MM/YYYY)] [w/WEBSITE] [t/TAG]...` +
-Shorthand: `a n/NAME [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [b/Birthday] [w/WEBSITE] [t/TAG]...` +
-//end::bday[]
+
+> ##Version 1.0 (Current)## +
+Format: `add [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...` +
+Shorthand: `a [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]...` +
+* All fields *MUST* be present except *TAGS*.
+
+> ##Version 2.0## +
+Format: `add [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [b/BIRTHDAY] [w/WEBSITE] [t/TAG]...` +
+Shorthand: `a [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [b/BIRTHDAY] [w/WEBSITE] [t/TAG]...` +
+Only NAME is compulsory +
+* Name field *MUST* be included. +
+* Other fields are optional.
+* Birthday field is in DD/MM/YYYY
 
 TIPS:
```

Comment:
I think can remove this line if there are no tips left

Reply:
which line?

Reply:
There are lines below TIPS. I'll mark this as resolved.